OloEditor/src/Panels/SceneHierarchyPanel.cpp

```
@@ -383,6 +615,176 @@ namespace OloEngine
         }
     }
 
+    static void DrawParticleEmissionSection(ParticleEmitter& emitter)
+    {
+        if (!ImGui::CollapsingHeader("Emission", ImGuiTreeNodeFlags_DefaultOpen))
```

Clang (14.0.6)

[warning] statement should be inside braces

(readability-braces-around-statements)